src/views/templates/template-details/TemplateDetailsView.scss

```diff
@@ -0,0 +1,123 @@
+@import 'styles/mixins';
+@import 'styles/palette';
+
+$padding: 20px 0;
+
+
+/* layout */
+.root {
```

Comment:
Rename this to templateDetailsView - sometime ago i talked about root classes which is convention from Material UI. After testing this - i think we should use normal class names which tells more. templateDetailsView will be good